lib/dokku_cli.rb

```diff
@@ -99,6 +99,7 @@ def git_config_match
     end
 
     def run_command(command)
+      command = command.gsub(/ --remote=[\S]*/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.